cogboard-webapp/src/components/Widget/styled.js

```diff
@@ -113,13 +113,13 @@ export const StyledCollapse = styled(
     <Collapse {...props} />
   )
 )`
-  bottom: 1px;
+  bottom: 2px;
   background-color: ${({ isDragging, status, theme }) =>
     !isDragging
       ? mapStatusToColor(status, theme)
       : theme.palette.background.paper};
   box-shadow: ${({ isExpanded }) =>
-    isExpanded ? '2px 4px 4px 2px rgba(0,0,0,0.3)' : 'none'};
+    isExpanded ? '4px 4px 4px rgba(0,0,0,0.3);' : 'none'};
```

Comment:
I don't think there shouldn't be a semicolon in `'4px 4px 4px rgba(0,0,0,0.3);'` - the semicolon after the braces should be sufficient.

Reply:
Good catch, thanks. Fixed in a8d3f80